Rakefile

```diff
@@ -1,8 +1,9 @@
 require 'tmpdir'
 
-desc 'Run unit tests on iOS 11.2'
+desc 'Run unit tests on Swift 4.0 & Swift 4.2'
 task :test do
-  sh "set -o pipefail && xcodebuild -workspace 'Bootstrap/Bootstrap.xcworkspace' -sdk 'iphonesimulator' -scheme 'Bootstrap' -destination 'name=iPhone 6,OS=11.2' build test | xcpretty --color --simple"
+  sh "set -o pipefail && xcodebuild -workspace 'Bootstrap/Bootstrap.xcworkspace' -sdk 'iphonesimulator' -scheme 'Bootstrap' -destination 'name=iPhone 8' SWIFT_VERSION=4.0 build test | xcpretty --color --simple"
+  sh "set -o pipefail && xcodebuild -workspace 'Bootstrap/Bootstrap.xcworkspace' -sdk 'iphonesimulator' -scheme 'Bootstrap' -destination 'name=iPhone 8' SWIFT_VERSION=4.2 build test | xcpretty --color --simple"
```

Comment:
We'll need to update these to look like this:

```sh
set -o pipefail && xcodebuild ... | second_curtain | xcpretty ...
```

Notice how it pipes the output through `second_curtain` _before_ it goes into `xcpretty`. This has to be there to work.

Reply:
Oh shoot, I meant to leave this on #165 oops.

Reply:
_cries hard_